tools/UpdatePaths/Scripts/72076_escapepodports.txt

```diff
@@ -0,0 +1 @@
+/obj/docking_port/stationary{roundstart_template=/datum/map_template/shuttle/escape_pod/default} : /obj/docking_port/stationary/escape_pod{@OLD;roundstart_template=@SKIP;height=@SKIP;width=@SKIP;dwidth=@SKIP;name=@SKIP}
```

Comment:
this works, but it seems the only thing you want to preserve seems to be the dir variable? You could just do

```suggestion
/obj/docking_port/stationary{roundstart_template=/datum/map_template/shuttle/escape_pod/default} : /obj/docking_port/stationary/escape_pod{dir=@OLD}
```

...if you know for a fact you want to discard everything (which is valid here since this is an abstract object that players don't interact with). Probably works even better to ensure that downstreams just don't get odd cruft? Just letting you know this is a thing anyways.